wrapanapi/systems/redfish.py

```diff
@@ -13,18 +15,54 @@ class RedfishSystem(System):
         hostname: The hostname of the system.
         username: The username to connect with.
         password: The password to connect with.
+        security_protocol: The security protocol to be used for connecting with
+            the API. Expected values: 'Non-SSL', 'SSL', 'SSL without validation'
     """
+    # Power states as defined in Redfish
+    RF_POWER_STATE_ON = 'On'
```

Comment:
Just to be sure: we need those variables defined as standalones to make them accesible from integration_tests, right? If not, I'd just provide the hash directly like

```python
POWER_STATE_MAPPING = {
  'On': 'on',
  'Off': 'off',
  ...
}
```

because we're polluting the code :) Alternatively we could put them in separate file say `wrapanapi/utils/const_redfish.py`.

Reply:
Yes, I foresee tests where we will want to check the status in API, not just in GUI.